### ISSUE TITLE
docker compose v2 multiple files  

### ISSUE THREAD

**Issue by navinSing** [docker/compose#2900]
have a multiple docker compose files; start with `docker-compose -f a.yml -f b.yml up -d` 

network is specified in first file a.yml with 

```
networks:
  nwfront:
    driver: overlay
```

how do i use this network in services of b.yml 

```
... # some service
   image : abc   
   networks:
      - nwfront
```

gives ... 
ERROR: 500 Internal Server Error: subnet sandbox join failed for "10.0.0.0/24": bridge creation in sandbox failed for subnet "10.0.0.0/24": invalid argument

also see .. but that's within default, what if i want to include multiple externals

```
networks:
  default:
    external:
      name: nwfront
```

// docker-compose v1.6.0

**Comment by dnephin** [comment]
There was a bug with multiple files that was fixed in #2851 (in master now, will be in 1.6.1) release.

But from your example, it's not clear to me if you're hitting this bug or something else. The failure sounds more like it hit an error in docker engine, not that it failed to find the network. If the merge had failed, you would have gotten an error message from Compose about it.

Are you able to reproduce this every time, or did this just happen once?  What version of Docker are you using? Could you try 1.10.1 ?

**Comment by navinSing** [comment]
will wait for compose 1.6.1
 was using engine 1.10.0, will wait for compose 1.6.1, before upgrading.
the compose was failing everytime ( not just once) when networks was used in different file with -f options defined in previous file.
```
networks:
      - nwfront
```
ERROR: 500 Internal Server Error: subnet sandbox join failed for "10.0.0.0/24": bridge creation in sandbox failed for subnet "10.0.0.0/24": invalid argument
